### PR TITLE
Fix README according to Alex@datadog's advice.

### PR DESCRIPTION
**Pull request by numno1**
### What does this PR do?

Fix README.

### Motivation

This fix is according to advice from Mr. Alex Vetras, Product Manager@datadog.

### Review checklist

- [x] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [x] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### CODE REVIEW

**Comment by hithwen** [comment]
Hallo, validation is failing with following errors:
```
     image: appkeeper/images/integration2.jpg is linked in its readme but does not exist
     image: appkeeper/images/synthetic_test2.jpg is linked in its readme but does not exist
```

The files are actually named `intergration2.jpg` and `syntetic_test2.jpg`. Please fix the file names.

File: appkeeper/README.md [hunk 1]
```diff
@@ -8,38 +8,83 @@ the need for expensive manual intervention.
...
 ## Setup
 
 ### Step 1. Get the SIOS AppKeeper API Key
 
 Get the SIOS AppKeeper API Key from AppKeeper GUI.
 
-![snapshot][2]
-
-### Step 2. Define the Webhook in the Datadog Integration Dashboard
+1. Click  the Account Information, and open the modal dialog
```
Comment (sarina-dd) [inline appkeeper/README.md:21]:
```suggestion
1. Click **Account Information**, and open the modal dialog
```

File: appkeeper/README.md [hunk 2]
```diff
@@ -8,38 +8,83 @@ the need for expensive manual intervention.
...
+2. Click Get Token
```
Comment (sarina-dd) [inline appkeeper/README.md:22]:
```suggestion
2. Click **Get Token**
```

File: appkeeper/README.md [hunk 3]
```diff
@@ -8,38 +8,83 @@ the need for expensive manual intervention.
...
+3. Copy the Token
```
Comment (sarina-dd) [inline appkeeper/README.md:23]:
```suggestion
3. Copy the token
```

File: appkeeper/README.md [hunk 4]
```diff
@@ -8,38 +8,83 @@ the need for expensive manual intervention.
...
 
 ![snapshot][3]
 
-### Step 3. Define the PAYLOAD and CUSTOM HEADERS.
+### Step 2. Define the Webhook in the Datadog Integration Dashboard
+
+1. Click the Integration
```
Comment (sarina-dd) [inline appkeeper/README.md:29]:
```suggestion
1. Click on the integration
```

File: appkeeper/README.md [hunk 5]
```diff
@@ -8,38 +8,83 @@ the need for expensive manual intervention.
...
+2. Click the Webhooks (*If you did not installed the Webhook, please install the Webhook.)
```
Comment (sarina-dd) [inline appkeeper/README.md:30]:
```suggestion
2. Click the webhooks (*If you did not installed the webhook, please install the webhook.)
```

File: appkeeper/README.md [hunk 6]
```diff
@@ -8,38 +8,83 @@ the need for expensive manual intervention.
...
 
 ![snapshot][4]
 
+### Step 3. Define the PAYLOAD and CUSTOM HEADERS
```
Comment (sarina-dd) [inline appkeeper/README.md:34]:
```suggestion
### Step 3. Define the payload and custom headers
```

File: appkeeper/README.md [hunk 7]
```diff
@@ -8,38 +8,83 @@ the need for expensive manual intervention.
...
 3. Register the AppKeeper API token in the **Custom Headers** "**appkeeper-integration-token**"
 
-For more information on the AppKeeper's Integration, review the Appkeeper [documentation][5].
+![snapshot][5]
+
+### Step 4. Integrate with Datadog monitoring
+
+As an example, create a new synthetic test and set up the integration with AppKeeper.
+
+1. UX Monitoring – Select Synthetic Test
```
Comment (sarina-dd) [inline appkeeper/README.md:46]:
```suggestion
1. Navigate to UX Monitoring. Select **Synthetic Test**.
```

File: appkeeper/README.md [hunk 8]
```diff
@@ -8,38 +8,83 @@ the need for expensive manual intervention.
...
+2. Click the New Test for creating the Synthetics as new
```
Comment (sarina-dd) [inline appkeeper/README.md:47]:
```suggestion
2. Click **New Test** to create a new Synthetic test
```

File: appkeeper/README.md [hunk 9]
```diff
@@ -8,38 +8,83 @@ the need for expensive manual intervention.
...
+
+![snapshot][6]
+
+3. Setting the fields of monitoring. It set for the Web page
```
Comment (sarina-dd) [inline appkeeper/README.md:51]:
```suggestion
3. Set the monitoring fields.
```

File: appkeeper/README.md [hunk 10]
```diff
@@ -8,38 +8,83 @@ the need for expensive manual intervention.
...
+
+![snapshot][7]
+
+4. Add the Webhook that set in Step 2 and Step 3 in the Notification setting(Notify your team)
```
Comment (sarina-dd) [inline appkeeper/README.md:55]:
```suggestion
4. Add the webhook that set in Step 2 and Step 3 in the Notification settings (Notify your team).
```

File: appkeeper/README.md [hunk 11]
```diff
@@ -8,38 +8,83 @@ the need for expensive manual intervention.
...
+
+![snapshot][8]
+
+5. Datadog has the function to suppress notifications when an alert is raised repeatedly.
```
Comment (sarina-dd) [inline appkeeper/README.md:59]:
```suggestion
5. Datadog has the functionality to suppress notifications when an alert is raised repeatedly.
```

File: appkeeper/README.md [hunk 12]
```diff
@@ -8,38 +8,83 @@ the need for expensive manual intervention.
...
+If you set it, AppKeeper's recovery API will not be called by Webhook. Please don't set it to inhibit.
```
Comment (sarina-dd) [inline appkeeper/README.md:60]:
```suggestion
If you set it, AppKeeper's recovery API will not be called by the webhook. **Do not set it to inhibit.**
```

**Review by sarina-dd — CHANGES_REQUESTED** [review]

**Comment by numno1** [comment]
Thank you for your review.
I fixed the indicated items. Review again, please.

**Comment by hithwen** [comment]
There are still some unadressed comments

**Comment by numno1** [comment]
@sarina-dd @hithwen 
I fixed the indicated items. Review again, please.

**Review by sarina-dd — APPROVED** [review]

**Pull request merged**